packages/cli/src/bin/run

```diff
@@ -1,6 +1,20 @@
 #!/usr/bin/env node
 
+const path = require('node:path');
+
 (async () => {
   const oclif = await import('@oclif/core');
   await oclif.execute({ development: false, dir: __dirname });
+
+  // Show deprecation warning if invoked via the old "zapier" binary name
+  const scriptPath = process.argv[1] || '';
+  const binName = path.basename(scriptPath, path.extname(scriptPath));
```

Comment:
Note that this doesn't work on Windows. When I was working on https://github.com/zapier/zapier-platform/pull/1181, I found on Windows you can't get the executable name in `process.argv`.

Reply:
@eliangcs Agreed, that is currently what I have in the [PR](https://github.com/zapier/zapier-platform/pull/1191/files#diff-a48d4a7103160e2c67a3dc346e2158c2104ed105a2e18f28675c2eb4e8286da3R11) ( my comment above was not the best at explaining this! )

Reply:
Good idea, I'll add that in and update the docs to mention that!

Reply:
@eliangcs I updated it so `ZAPIER_SUPPRESS_DEPRECATION_WARNING` can suppress this warning for all OS!